Scripts/GenerateSampleViewSourceCode.swift

```diff
@@ -71,24 +73,28 @@ let templateURL = URL(fileURLWithPath: arguments[2], isDirectory: false)
 let outputFileURL = URL(fileURLWithPath: arguments[3], isDirectory: false)
 
 private let sampleMetadata: [SampleMetadata] = {
+    // Finds all subdirectories under the root samples directory.
```

Comment:
This comment doesn't really make sense here. I saw the comment and then saw right after that line we are creating a decoder and I don't see how they go together.

Reply:
Good point. No idea why I added it in the first place. 🤦